apps/backoffice-v2/src/lib/blocks/components/KycBlock/hooks/useKycBlock/useKycBlock.tsx

```diff
@@ -317,7 +319,7 @@ export const useKycBlock = ({
         .buildFlat();
     }
 
-    if (status === 'approved') {
+    if (status === INDIVIDUAL_KYC_CHECK_STATUS_ENUM.approved) {
```

Comment:
Please stick to enum conventions.
```suggestion
    if (status === IndividualKycCheckStatus.APPROVED) {
```

Reply:
Fixed